spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/DefaultSource.scala

```diff
@@ -0,0 +1,93 @@
+package org.apache.spark.sql.cassandra
+
+import org.apache.spark.sql.{DataFrame, SaveMode, SQLContext}
+import org.apache.spark.sql.SaveMode._
+import org.apache.spark.sql.sources.{CreatableRelationProvider, SchemaRelationProvider, BaseRelation, RelationProvider}
+import org.apache.spark.sql.types.{DataType, StructType}
+
+
+class DefaultSource extends RelationProvider with SchemaRelationProvider with CreatableRelationProvider {
```

Comment:
No usages found in project. Needs documenting how it gets instantiated and where it is configured.